concept/type_/attribute_type.rs

```diff
@@ -141,6 +177,15 @@ impl<'a> ThingTypeAPI<'a> for AttributeType<'a> {
 }
 
 impl<'a> AttributeType<'a> {
+    pub fn get_value_type_declared(
+        // TODO: add concept api tests
```

Comment:
TODO still relevant?

Reply:
Yep, specifically for this call. Probably just need several steps in existing tests

Reply:
Fixed